std/markdown.joke

```diff
@@ -4,7 +4,14 @@
   markdown)
 
 (defn ^String convert-string
-  "Returns the HTML rendering of Markdown string s"
+  "Returns the HTML rendering of Markdown string s `opts` is an optional map of boolean rendering options (all default to tru to `true`)
+
+  :with-hard-wraps? - Render newlines as <br>.
+  :with-xhtml? - Render as XHTML.
+  :with-unsafe? - When false, all raw html will be omitted from the output. When true html is passed through unchanged.
+  "
   {:added "1.0"
-  :go "convertString(s)"}
-  [^String s])
\ No newline at end of file
+   :go {1 "convertString(s)"
+        2 "convertStringOpts(s, opts)"}}
+   ([^String s])
+   ([^String s ^Object opts]))
```

Comment:
You can specify `Map` type directly here, which will simplify the implementation: `[^String s ^Map opts]`. See [joker.http/send](https://github.com/candid82/joker/blob/master/std/http.joke#L22) for example.

Reply:
Thanks! I probably should have looked harder for an example of that